ppa/archive/models.py

```diff
@@ -121,6 +122,76 @@ def stats():
         return stats
 
 
+class ProtectedFlags(Flags):
+    '''Flag set of fields that should be protected if edited in the admin.'''
+    title = ()
+    subtitle = ()
+    sort_title = ()
+    enumcron = ()
+    author = ()
+    pub_place = ()
+    publisher = ()
+    pub_date = ()
+
+    @classmethod
+    def deconstruct(cls):
+        '''Give Django information needed to make
+        :class:`ProtectedField.no_flags` default in migration.'''
+        # (import path, [args], kwargs)
+        return ('ppa.archive.models.ProtectedFlags', ['no_flags'], {})
+
+    @classmethod
+    def all_fields(cls):
+        return list(ProtectedFlags.__members__.keys())
+
+    def as_list(self):
```

Comment:
Can't you just use the existing flag `__iter__` method?

Reply:
Likewise for this! Much simpler even if I have to do a single dot split (shouldn't be too punishing).